### DISCUSSION TITLE
Workflow diagram (flowchart)

### DISCUSSION THREAD

**Discussion by talmurshidi** [HackYourFutureBelgium/workflows#2] [Ideas]
## Motivation
I have noticed that the learners are struggling of understanding the workflow of starting an issue till it is solved, maybe the problem they need diagrams and flowchart along with the theory explanation of Workflow Module, so I decided to draw a flowchart for workflow module. I would like to make the self-learning is easy for the learners. I would like to thank [Evan Cole](https://github.com/colevandersWands) @colevandersWands for the inspiration.

### Definition
> A workflow consists of repeatable activities necessary to complete a task. [https://fluix.io/](https://fluix.io/help/workflows-and-processes#a_2500118779338141)

## Feedback
I appreciate your feedback to improve and enhance the flowchart. Also, you can correct English notes. You will find the images of workflow flowcharts along with the source to make it easy for you to update the flowchart to illustrate your idea then you can share a new link.

### Abstract workflow diagram
> It is an overview and a big picture of workflow.

[Abstract workflow diagram source](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&title=HYF%20abstract%20workflow.drawio#R5VpbV6M6FP41XUsf6grXto%2B2enTmqMdRzzhz3lJICyMQDMG28%2BsnlHBJoZReoB7tS8nOlb339%2B1kh44ycudXBPrWLTaR05GBOe8oFx1ZloEus79IsoglPbUXC6bENmORlAke7d%2BICwGXhraJAqEhxdihti8KDex5yKCCDBKCZ2KzCXbEWX04RQXBowGdovTZNqkVS%2FtyL5NfI3tqJTNL%2BiCucWHSmL9JYEETz3Ii5bKjjAjGNH5y5yPkRMpL9PL8ZfHs3LzoV1%2B%2FBa%2Fw3%2BHfT3ffu%2FFgf23TJX0Fgjy689DDN%2Fr9QtP%2Fkx7Qtf34pDw%2BXP9Ohn6DTsj11ZF1h00yHLOHafSQCCaYTc60QBdctfpriJOKbrA0%2FDlrIOn%2BPO7G69cMZGAHE2GgjqxMlr9i3wfkYorStZFM7uPAppgsinUnVza9DsenSQ1TTTy1uJysw4aGTJxTiizoQiY49EwUqVpi1TPLpujRh0ZUO2PAYjKLug6vntiOM0pfXwFgMFAvmDygBL%2BgXE1fHiu6ns630QMScyJC0Tzn%2F9wjrhB2EY10BXhtVwLcuzm8%2B7w4y7AiqVxm5XCStIMcntN06MwF2QP3wi08Ul7rkdv5zQEcNbX2DWZ8wnpUettPHLICGGHXDykip5X%2BVUBXwecO6l8AjEZtepGmDY7sRlKvoEJkssDAi5hQC0%2BxB53LTDrMlAxYKWtzg7HPVfsLUbrgUQ6GFIuKZ3olix9R%2FzMtKf7kwy0LF3OhtOClGPhJfJK3NlXAfM%2Fgrzn%2F58etp98v7gfhWA%2B%2F3oLX4TceGgCFZIpohdrUuF2kqkrDE%2BRAar%2BJMfbgRlQ2ccFmjKtrMO5C29see2Az9gocvsLuuxu5Ph4loG4kdVCCRrUxNPar9hklZhxjYiLS5Vx%2FvuxKTrrdvPw05vnaFkfM91mD0DchZdaMdniRDwAnYndHoPS9VrQ6MUEBdt5QvMedOLZBg8bm8mEQ2N40mstCxktQHnKOSYwCLWYs2QIxVhHeRmLUahJjEhHBGUhPEhyFCi%2FW5k4%2B%2Bj22l2SXNMGTScDWuorJdBG7w1T9GHzLdzvH41ttULL9aZVwByWWbB%2F1jaNXq4ne%2Frva1mjNwYz9QzdCCA8v0DOsVLbjgeN%2FAkJVXTmDlIFQahOEetWm58DJlS93T5cPd5dP%2B%2BQwkGeeRxk3VvKwFzGACQMrPXTm7C06x7423gjLnAW1EgMmsl0DK3egnug%2FhbNpzEa8Uz7RtjKONtgwUExXhYEOFcWlsqTeByT%2Ffk3yTzD%2FTth%2F%2FVFob%2FaHpvl56F06Nr1L69MT5Yfaumb0cCNnRTCGxst0afh8czIdnzBtdmSmBZB%2FKD9hJssJfOiVL%2BgQs1ScY%2BOJtwlrzXPdr9D1kyl46Exvi1o5xkp1E3yS%2FK7IMFl3E2xoYNe16achRFU5OiGqx8Deupx7Wnes1FICtc2YrJtc2nODqw40wWMU0PLGVG4O7H4YWFvc6H408CtySYq%2FXfCvP%2B0eIqshHpvvH4q2Xr9r2iY1Xs9f2qS1tPBOMuZS3aSbUpPVuEuDM6D0FDGk7Xmwbz5jLn2KXF7rF5iqKkaqNi8wS%2B%2FPyyLX%2B9nqtPl5QRXU85RQ9RlC0xsdSRPvYyR5x1Te6kBpTvnwO6ZSfTX4AUQhpvrhcj35i7oseLo4oEu7GIh%2FhGWifQPoDRwjR3Rw6NhTjz1HsyBGP8OIMmwDOue8wrVNM8YSYu8Ix8vxIhf3IxMsjaINO9pFqYtXInuVm9IvUvkknfxHn2uufLVBT%2BStBED7%2BrMYFWWx%2F0GiWxVcm9mvR9N%2F0N356n1w6e78QPGLFbNPkGPDZx9yK5d%2FAA%3D%3D)

![HYF abstract workflow](https://user-images.githubusercontent.com/34141278/232173668-e8458256-69e3-48d8-9081-2a6d779caeda.jpg)

### Detailed workflow diagram
> Detailed workflow process on remote repository and local repository.

[Detailed workflow diagram source](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&title=HYF%20Workflows.drawio#R7V1rc5s4F%2F41nmk%2F2MMd%2B2PiNK13kzZNsm%2BafulgW7bZgHEBN8n%2B%2BlcCZEASNyMuTrsz3diyECCd5zlH5xxJA3lqv3x0jd3m2lkCayAJy5eBfDGQJFkQBPgHlbyGJdJkLIcla9dchmViXHBn%2FgeiwujC9d5cAi9V0Xccyzd36cKFs92ChZ8qM1zXeU5XWzlW%2Bq47Yw2ogruFYdGlD%2BbS34SlY0mPyz8Bc73Bdxa1SfiLbeDK0Zt4G2PpPCeK5A8Deeo6jh9%2Bsl%2BmwEK9h%2FvlYfb6YF09aR%2F%2F%2Bur9NP45%2F%2Fv%2B8%2F%2BGYWOXVS45vIILtv7RTava%2Ffmj7n%2B7cs3X2f0G7Id3j0MpbPqXYe2j%2Fore1X%2FFHQiWsD%2Bjr47rb5y1szWsD3Hpuevst0uAbiPAb3GdK8fZwUIRFv4LfP81Eg5j7zuwaOPbVvRryfeL%2BsFz9u4C5LxUNF6%2B4a6Bn1Mvag%2B9YEJWot77CBwb%2BO4rrOACy%2FDNX2mBMiK5XB%2FqxX0PP0TdX2EoBGoo7u7Pbu%2Bp8fB813k6yLKU7kkooztUz35ZIzyPVpbzvNgYrj%2FyYHf4P1Cd543pg7udEfThM6xWeQR%2BAdcHL7l9hglEiXAW0cdQVaP3fI7BKGKS2SSAqAkN9bPahYSDF9P%2Flvj8iJoaqdG3i5eo5eDLK%2F6yha%2F7LfklcRX6Gl8WfMPXcUYTJu0iNCm9QhOW1BhNF2BlbgG61wDRt2bBlzlfObCPkPKxHDeopv3cIz6HnSivVkKg%2FuIibY3%2BerYR3B%2B3AZ8vbCb8eec6cwvYlJSxYJdELgVr1CgWS6EpgKoiCVCNAVCJAdBxUwDV%2FwC0EkCVkgDVegVQhQLolx3Yojtta%2BLT9Lw9yMang9r%2FaPqf9vP6GA1LpofHgwbzZKJcNKhRdaF3gBVFqiPbQCxGn1gNfRjpQgWkH4%2FY2XdJ3nivwuK7DsCPn4u%2Flj9%2BDsVxWQtV4I3Z6NIbxwyAhUVOHBNSpYiEtITPGl1HCMzhQUrJELNLZIoOupApzsys0cPMfHmpV8ysUUNxtlwG03b0vw2i1pBi5TNMsXMXs2sd4r43fTiMEnxq4QJ4C9fc%2BSZia1RwZcyB5YWfr00LeOinqPKN6%2FwLFr6XQfinyPITYVKG5dUWWV6i5eKz89aJH8lRYtyn00ByGRJx%2BCWw2sNHFzXCii%2FryCk9NSvt6SjLMJH4DYXRQbiwBEYPVVvTaIRky8JkpKabCV%2Bc0jV0YxIJE0ZjGYorbgxXdFYrD9RVbp%2FOv13vhp9mL0%2F30%2BHTg%2Fj9ylgzJqMETcYAwpyIfhh6gRxBlhVEbfcSMiXBmY%2BgNO21OIuKYfWYRFX%2BLKoinEsAjRD3LpAnycVAm4z5QCt9ARdxzn35hDzPvINt4IIdMhQWFtTPsJVLSg7djWPP916GJ7KqIhaE6bRJRawShrGkKyNVpHUxy4c5kZvSxZ1Yx8co0GPRzRuvUllN2TM3Jh2gmVqB4ZtCG08rfG36GL6hE8WwES6D9tDdhnvXOpSm7taG3d0w3HWZhLs2GcljGu5ymw4WiXaWnZLhzRvLZR2eOMbbFyzTozjben4YTNjubL4wDhoUTHyDN4NQZUyY%2FFkIbdUFKo07BWQCg48pCGabzokxu7yM5q7tukZz7OXTA7dEOzD%2F2S0NPxmIOAbGtmFu0QMZng%2FcTBgLc9fYLjbYSQdfGHh%2BCCsvcKHxNxB2%2BzdFK6oupWlFkRgON61NTpHpaVYXJFNIFqm5vtgqWehlyWLSL7LQGSNbDEhBYANy6oKQaFBmwxY8J%2FggA58kHYSQPpBI2ugPi4dbwwak3V8X51my1T7%2BJTIPQsJ%2Byu7w362Rz8mmSA%2FlEqyMfSBVfaCPSUn6kPuVKYifmxN93D2bfmw7hG6FGiyy2IDFk7P3m%2BORfFFL8YtpWbTorV1jaYK4jS1ydXRDOoTRIbKMDr1V0qGjfF2wUBZrEEMfifRJp29hcilmIbVXLISfmxcLofxk9JzbZXCpvbOAH%2Fs5fcN74jTdKE8fvSUKWR3h8Gx3TNFJnmZ%2B4D2PGo6KSQojvbIVFAch00FKpQHuUDuzYNgxSIXM9xdxumJBOP7MdY3XRLUdquBl30gl0yBFYvULUV9WhLz68EP4BFwDpdgtyaDIOWavz18StDbP5DQ0cEEOVBoOhmWut%2FDzAsoTgJJ%2FjjjFXBjWWfSDbS6XIfKAZ%2F5nzKNscvk86mDYuHo%2BUC9oAGVmtNAynMsPFMsd1npFTzNILqfKCJJLIpHpwScbZShqhFSkW2guhq70w7lT1c5qM7upPLn1y7uj0IZRkKXTOaKPHcV8AeYCb40PnJW24Itlk5Pxe2Eio3cG%2F63MrelFPjhiep0whFFWTTlbuHSyzQlaxaSxIclMq5gVCBQFJVu46qG%2FH5nsJYn9MMy8ubusa02ResXdMl%2FX2oWTQDIB1amzRHj%2BuoeM76M3nCLHGa0n3h5uNQK3GbPZVgP4Ci9Cfzu5vEdm%2B1RlnmPm522aoZihiqmsbOpg0u7BQZ6als94koaUqEzSTbSbBc%2FuH3o23PECkoqO39K%2Bp04Wg5SXU%2B6ZM%2B2LksRVTZ%2B5kaPZzVk3bMGXM7frQbl8nqxbwZqZd1i5jm1wuAdwXcf1%2FkwSQmYkEvhFdcSwNWSZMUcQqbVE%2FMwNrVviqxbbPzYa12EeIVbGHTiP2Bpa1ws0dMaiN26UycohP55jLs2XAY7NRXzz20bndHKN0JgRxWfNZtSmZjO44VO0qzLp5WBVHRPs500vZTMPlZboRZTwTAI7w8Yt8wvfnMZoy4GAX1Zovf8gyk%2FyfGMdmkhwLI2KuUlG0Go6LQm13mpSUm94SxR0FcW3S%2Fhi9DZ3GtC6Za%2B2%2FB2tsBdzvxGGbcSs19kSC%2BbTFHroij1vSobn7c4HuwPD7Ml1G3yWTSz2bjB8JdMq%2B%2BAB7D5FV2HEEfJEuqzvTRgJIrmQDGdP9djtwTc6EerYhWPbgXK0geehzXQrpvviy4c2pVrjFn8zxSqLpSIcTS0n0Gdfr9fu7Ofry6Pvf76d3Dw%2FfmNuQtI%2Bp3DmB5WxXJD59tw3Uqs3P2MFiY8H8s0%2BSBkIreXFxtiucyJJbBjvgiaaQKbYJ2TKWhqYKp66dxZ41Gi%2FTFa20GlP32sl5LYZSNHKJuRrcjWjY4isDrVmwK95K0PrzUZvR9iffFzcfYvtlRZJtaJIQokUcOi4xxLJd6kJO9wXa0QbuOtAkzrblWWijSVPK5RWPoW0LTUrSxor6oZN0FTUrbl9TbvZ%2BT%2BDkMRKhNSMVuXNUmX37mgr5qaRPnFBSzfRsE9c42vm3znWr3hFXMxOv23gTZOImOqk6z07NL5RkA%2BfL8qObiPHwlSNzFVab92UVIiyNk6JhS4wxKLVM2ewa7GJHWAfvtz%2BfXn15QHWubn9Mv1wd1dWZGB%2F%2BoOcrPFonJKDGhWVtzBYUpXWqWUWqaiUfQ2lyjI8DytLfDRXda1WIcxPZHDAUWXYNCJDrpSmfHpytxZNNT9B1kAfb4XkefoKPYISbxuk1kgWEkQlvREdpZJBBDe3ddqG9VNHqPTW%2BGh4i35VLeFKVNr08Ct9mt8UOVyCbzfANeG7IxXSEEWwDxsp7eFrOlUaJ6mO0xbLWCJkhN%2FpK3lcyDd3J95NCBJOfFZIYUCCPAYkcQoIXfnM86DFAQB9SsiVuX2Cf2bhaVBvno5I64RJR2pDdMREWbZKm5ODX9sILpeYgf6jr70FtuMDWrRuwc7xTN9BXUH%2B9i7Ufu9LC3Vxtkf23g8xR2fN05Kim7DWk3JHSeRYmssaez8HTstzJdJeZsXAm9KQ7IO3juI5WnA4SOphuK8cOG%2BCV%2BSK2yNUabDO1LF3e6gu3%2BcKGAWvQqd1LQFrPEtR650c8c2lT2%2FAR%2Bz4W3sTPqK97hVh68k3CkMTNuUVZIoLK22rfUudsxHNyIBn1hvzNqFrDQXtn%2F2zTfcJbdMtCvphUV4uvKVRQxnrbLuiH8l1PUgtZ5xsyu6xlk42nehySlo0gWiCXwgwrz8SgpHW9llZ4VzP8jliR943wxhjqYQ1MG51EtINN7R%2BeE8u7otdb53tuZ373AkgX0dpQqTBDpmkLWTb7Ed4M%2BAlfexM8E5aBW8%2Ftvfqyd7VnR1wzVb2qialxGWoStJh4UXVw2VJxQEbk6nGGj4VXaJJx0mkAIXHcIeWRKS%2Bt3Nvl%2BuAPIZs7p2lE16zt7dt0kvDPnNRFMjZRMZB2%2BM2KYaVQ%2FQ7UEweBIonFNx3x2fTgj4mBQYPT%2BXTq2Vys2yqqabphcOS2iw%2F92enLFH0OAmXGaSuvlSgRytecrFTrMerb3EniHhrxV6sL8h9%2FwQOHszgslkgqsHJFNR2vMFuFKmU3FAf1z%2FyOjt%2Bl9SXjWlFmbJ8xhqtE1mxZFGWsoe1llKUWNHk05lF98AzhxFXrEm5e%2Bwz9J9AnuGgTAib6nhVSjXVsCoVaUffEaY6Q9%2FOHXcJ3GFkr58Ft3DfDYfJ8vdMDVyoyo9putrEYYYSBW9cZ%2B0Cz2tmBtExV4qyQG6hM5S08Qi7mdtYV8yGe%2BHakze3UXHVDMQjlpf20yor6yqNrTJRkJV6VtkhvkLaCiUDLDSHi%2Fq4oKl2NzHO7evKKUQZqZJB%2BuIgM1XbfvWAtRqksinNYzIae0GWMt63N2ZKbYQXyRUxpa5T3jd%2BZEkn9ARLr4g%2BbmSNVXO9rRK9rYuMvKmm1kGx%2B5l1INbpuLZaylnPS9EuVgZaW9Y71iA44RjvI1CZ%2BCUydZlsiZ%2Ft%2Fun82%2FVu%2BGn28nQ%2FHT49iN%2BvjPXwuFW6iwVJ6W%2FGW8Zx341qy9wr7q9ByDjPyTIjspVzsGJddFU9%2FVAUcHgcL3GV8o8%2FlIjzIYj6g7rHH%2BZ2Iqc02Zl3MIHC02AZy0HwubHLJrbsyLahaqfJNmYEkD4WVVVYO3EIOjNfril7Sz7tKHqna9fKbt5a%2BshXPCtMM4qoRt9rb0yNLczadoIqH2cnMI79VNPPJMpES81NIDkfm53tZMxYLtCM71FI1HHX83dCuAYP%2F%2Bmtg%2FIWGEuEgcvgXJZb8MsEz4PW%2FJT9VRuymvbFtLpPIhs3tDFxYvskVtIabToaWcZuru6pcsLZYT%2Bv%2FugRcmFidT3Sgprgu8Ss5pLskBiBW%2Fl0ylPkPnFCmszNcR%2F86jpopGLZQc7Da2eJzoX%2B8H8%3D)

![HYF Workflows](https://user-images.githubusercontent.com/34141278/232174163-83385cf9-69cc-493e-97d1-ec62bf84611d.jpg)

# Thank you in advanced

**Comment by talmurshidi** [comment]
[Updated diagram](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&title=HYF%20Workflow%20Steps%20v1.0.drawio#R7V1bc9u2Ev41mmkfpOGd0qMtx4laO3Fs9zjOS4eSYIk1LwpJxXZ%2F%2FQFIQiQBkOIFvMhpZ1JLEAiSwH7fLnYXwEie268fPWO3vXbXwBpJwvp1JF%2BMJEkU5Cn8g0re4hJ9JkYlG89cx2VJwZ35L8CXxqV7cw38TMXAda3A3GULV67jgFWQKTM8z33JVntyrexdd8YGUAV3K8OiSx%2FMdbCNSqeSnpR%2FAuZmi%2B8sarPoF9vAleM38bfG2n1JFckfRvLcc90g%2BmS%2FzoGFeg%2F3y8Pi7cG6etY%2B%2FvHV%2F2H8df7n%2Fef%2FjaPGLqtccngFDzhB7aZV7f78UQ%2B%2BXXnm2%2BJ%2BC%2Fbju8exFDX907D2cX%2FF7xq84Q4Ea9if8VfXC7buxnUM60NSeu65e2cN0G0E%2BC2pc%2BW6O1gowsJ%2FQBC8xcJh7AMXFm0D24p%2FLfl%2BcT%2F47t5bgYKXiscrMLwNCArqxe2hF0zJStx7H4Frg8B7gxU8YBmB%2BTMrUEYsl5tDvaTv4Ye4%2BysMhUANxd392e09NR5%2B4LnPB1mWsj0JZXSH6tmvG4TnyZPlvqy2hhdMfNgdwd%2BozsvWDMDdzgj78AVWqzwCP4EXgNfCPot%2FlZUYZzF9jFU1fs%2BXBIyiEJdtU0DUhJb6We1DwsGrGXxLfX5ETU3U%2BNvFa9xy%2BOUNf3Hg635Lf0ldhb4ml4Xf8HWc0YRJ%2BxialEGhCUtqgqYL8GQ6AN1rhOhbs%2BDLnD%2B5sI%2BQ8rFcL6ym%2FdgjPoedKD89CQKSzKRI26C%2Fvm2E98dtwOeLmol%2B3nnu0gI2JWUs2KWRS8EaNYrFUmgLoKpIAlRjAFRiAHTaFkD1%2FwBaCaBKSYBqgwKoQgH0yw446E5OQ3yavr8H%2Bfh0UfsfzeDTftkco1HJ%2FPB4siDMZspFixpVFwYHWFGkOrILxGL0idXQh5EuVEB6fcQuvkvy1n8TVt91AP7%2Bsfpj%2FfePsTgta6EKvDEbX3rjmiGwsMiJU0KqFJGQluhZ4%2BsIgTk8SCkZYnaJTNFBHzLFmZk1epiZLy8Nipk1aijO1utw2o7%2Bt0XUGlGsfIYpdulhdm1C3PdmAIdRgk8tXAB%2F5Zm7wERsjQqujCWw%2FOjztWkBH%2F0UV77x3H%2FAKvBzCP8UWX4mzMqwvNohy0u0XHx23zvxIzlKjft8HkouQyIOv4RWe%2FTookZY8WUdOaWnZqU9HWUZJha%2FsTA5CBeWwPihGmsajZBsWZhN1Gwz0YtTuoZuTCJhwmgsR3EljeGK7tOTD5oqt0%2Fn365340%2BL1%2Bf7%2Bfj5Qfx%2BZWwYk1GCJhMAYU5EP4z9UI4gywqitnuNmJLgzEdQmvY6nEUlsHpMo6p4FlURzoMEmiRXBpqqYHTWhRZXAS58%2F5QEL%2FyDNeCBHTINVhbUyLCVS0ryvK1rL%2Fd%2Bju%2BxquoVhPm8TdWrEqawpCsTVaS1L8trOZPb0r692MN1VGZdPPOGrFRWNw7McUmHZOZWaOpm0MbT7t6YAYZv5DYxbITLsD10t%2FHesw6lmbt1YWm3DHddJuGuzSY46pmGu9ylS0Wi3WOnZGrzxnJZFyeO6g4Fy%2FQoLhw%2FiMIHzs7mC%2BOwQcHEN3g3CFWmhJGfh9BOnZ7StFdApjD4mIFgvrGcGrPLy3i22q0zlDk5Uk8T2xLtsbx7MYPVNvaUxerSWfq7hri2DdNBj2j4AfBycS0sPcNBd2%2FBMtiC1bO7Rx%2FLPcvpcYw2E7Ico0gMf5uud0gwMj3n6oNxjjJHZqovtsQchYxwnDpmw6IOnTGyx0EqCGyQzj1gBDixwQEvFBdQ%2BCQpIoI5voiYAUTFY8ewATkJaIrzPNnqHv8SmQYhYTdlGv9alwaG3K%2FFz8nAyA7lGjwZ%2B1CqhkAfs5L0IQ8rURA%2FNyf6yFgtsY%2BhAYukjIWWeKRY1DL8YloWLXobz1ibIGnDQX6PfkhHypKOyDQ6OiUdOsjXBwvlsQYx9LFIn3T2FiaX4yykDoqF8HPzYiGUnoye01mHl9o7CwSJ0zMw%2FGdO043y9DFYopDVCY7O9scUvaRpFsfdi6ihVkhSmOiVraAkBpmNUSotcEdZ3wl%2FC4YdkFfIdH8RZyseicafeZ7xlqq2QxX8%2FBupZBakSCx%2BIerLilBUH36InoBr1BT7KBkUucTs9flLitaWuZyGBi5MgcrCwbDMjQM%2Fr6A8ASj554hTzJVhncU%2F2OZ6HSEP%2BOa%2FxjJOJpfP4w6GjavnI%2FWCBlBuQgstw4X8QLHcYalX%2FDSj9GqqnBi5JBKJHnySUcaiRkhFtoX2AurKMJw7Ve2sLnMuypPbsLw7Cm0YhUk6vSO67igWCzAXeGt84Kx0BV8sm5yM3wsTGb0L%2BO%2FJdEw%2F9sER0%2BuUIYxSbMrZwqUzb07QKiaNDUlmWsWsqKAoKPnC1Qz9w0hkL0nsh2Hmzd1lXWuKNCjulvm61i7cFJIJqM7dNcLz1z1k%2FAC94Rw5zmg98f5wqxG4zZnNdhrNV3gR%2BvtJ5a2Z%2BlOVeerMz7s0QzFDHaeysnmEabsHB3kaWj7TWRZSojLLNtFtEjy7f%2BjZcM%2FrRyo6fkv7nnpZC1JeTrnn0XQvShJXNX3mxY5mr2DZsAVfznQ2qQp1bgVr5t7hyfVsg8M9gOe5nv%2FfJCFiRiKbX1QnDFtDlhlzBJFaSsTP3ND6Jb5qsf260bhukgoLlXEPziO2htb1Ixo6Z80bN8pkJZTX55hL83WEY3Mx3%2Fyy0TmdXDA0ZUTxWbMZta3ZDG74FO2qXHo5WFV1gv286aVs5qHSEb2IEp5JYGfYtGN%2B4ZvTGO84EPLLE1ruP4rzk%2FzA2EQmEhxLo2JukhG2mk1LQq13mpQ0GN4SBV1F8e0Svhi9y40GtH7Zqyt%2FRyfsxdxuhGEbMev1tuCC%2BTRHPXTHPW9KjuftLgC7A8Psd%2BsonZrvUorV3guHr2Ra5RA8gP2n6CqMOEKRSFfwvckqbhybbVos7wP2e%2FANT0RKduXadqgdbeD7aDPdivm%2B%2BPKxTenWpMVfTLPKYqkQR1vrCfTF1%2BuNt%2Fjx9voYBJ9vZzcvj9%2BYm5B0TyqcCUKVaYJgvj33jdSaTdBYUeL6QL7ZhzkDkbm82hrOpiCUxIbxLmyiDWSKQ0KmrGWBqeK5e2%2BRR412zOSlC532%2FL1RRm6XkRStbEa%2BVmezn9msmZXxxrygPZtDG8y2bzXMUT4e76GF%2BkoLqFpRQIWJIMwGbwVrLEuGd%2FQv0Y828DahXnWdJ8tE20yeVmStfEZpV0pXljRWEA4bpJkgXHu7nPZzDkAOIYmVCKkdHcubpRimea8hOI10kQtatomWXeQaX6P%2FzrV%2BJgvkEnb6ZeNwmkSEWGeMOFynS%2Fg1vkGRD58vyo5uK4fEVA3UVVp%2B3ZZUiLI2zYiFLjDEotMTaPA6zTb2g334cvvn5dWXB1jn7v7DzV1ZgYG9GYwKUsjjUUoPaVxU3r5gyVRWo5ZZsaJS1jWUKcvwfawq8TFd1XVahZg%2Fkc4Bx5Rh0YgMqVLa8u%2FJ%2Fdoz1XwGeQNd3wYp8vod9Q5KvC2QRiN5lB4qaY34WJUcIri5bdI2rJ85TmWwpkfL2%2FWragm3otKlt18Z0uzmmLsl%2FHYDPBO%2BO1IhLVEE%2B%2BCR0t6%2BtvOmccbqNGuvTCVCRvidxFLEhXwTeZKthSDhJOeGHA1OkEeCpE4EoSuf%2BT60OACgTwy5Mp1n%2BGcRnQz17umItE6YdKS2REdMlOWrtCU5%2BI1N4HJZGug%2F%2BtpbYLsBoEXrFuxc3wxc1BXkb79F2u%2F30kJ9PPUjfyOIhKPzZmlp0U1Z62m5oyRyKi1ljb25A6e1uhJpL7Pi4W1pSPYhXLV4jhYcDpJ6GO4rF86b4BWF4vYIVRqsM3ft3R6qy98LBYyC11GXdSMBaz1lURucHPFNrM%2Fuxkdst9t4Rz6ivf4VYeeJOApDE7blE2SKCyuFq3tLnbMRzUiHZ9ab8jahGw0F7Z39i8z97GC%2F7gjp8H2BH0TC7Yc2Lv99vHf7d3UugCjohxV6hfCWJi2lr7PtimEk2g0gz5xxyim7xzo65XSmyxlp0QSiCX4BwKL%2ByDsxQMhLEW9pL%2F9srm7B9rzvhjGmUglrYNrpJKQfbuj8WJ9C3B93vfW2AXfhc6eAfB0nCZEGO2SSrpBtsx%2Fh3YCX9LEzwTvrFLzD2OtrIBtZ93bYNVvZq5qUEZexKkmHRRhVD5olFQdsTKYaa%2FmEdIkmHTeVABQdyV109hCnQ7rdtRtds7edLumlZZ%2B5KArkbCLn0O1plxTDyiD6FSimCALHJxTct8pn04I%2BJQUGD0%2Flk6xlcudsqqm26YXD%2Bto8P%2FdntyxRDDgFlxmkrr5QYECrXwqxc1yPV19zK4h4n8VBrC4ofP8UDh7M8LJFKKrhMRXU3rzh1hSZhNxIHzc%2FDDs%2FfpfWl61pRZmyfKYarRNZsWRRlvKHtZFSlFjR5NOZRQ%2FAM4cRd1yTcvfY5%2Bg%2FgTzQQZkRNlV9VUo11bIqFWlHXw1TnaFvl663Bt44ttfPwlt4v43H6fLfmRr4qCqv03S1icMCJQreeO7GA77fzgyiZ64UZYHcT2csadMJdjN3scaYDfejK0%2Fe3a7FVTMQaywuHaZVVtZVmlhloiArzayyQ3yFtBVKBlhoDhf16ZGmut3RuLCvK6cQ5aRKhumLo9xUbfvNB9bTKJNNadbJaBwEWcp4E9%2BEKbUJXiJ3jCl1nfK%2B8SNLOqEnXHhF9HErK6za622V6G1dZORNtbUKit3PrNOxTse11VHOelGK9nFloHVlvWMNghOO8S4ClYlfIlOXyZb42e6fzr9d78afFq%2FP9%2FPx84P4%2FcrYjOut0V2tSEp%2FN94yjrtuVFvk3sHuGoWg4XeoYmJqzWRBrmNqVT0eURRwyBwvepWKz0eUiAMkiPqjzPmInWxGUzg2nJJtF%2F7BkIoOmGUsKsFH0a7b2PYj3xJrnGzbmilBempUVWHt5iHozKy7tqw2%2BbRj8b2ugCu7H2xlwstykKiKdbiPYSNgO7WxtaHK9awNxkmiavaZRJloqT1S5HwSd76rMmfRQTseTCFVx9ssfxOilXz4z2DdnLfAWCMMXIZHvdyCnyZ4GXXm7Ryu2pDVrEen050X2bihjYkT23mxktbo24Yu1D1VDk077Ak2HD1CLm%2Bsrkc6UBN8F6o1XNgdESPwKh94eYrcJ85Ik7lD7mO6OQaWVlYxgj6MsxWYTgGJsesdcwQ6SlHTyAWzMilTZUmOdBFAki93yCQv1xydmPPfcr0Tyt9XlKy2HSuY8bpYi8uUKDoCsQgfJ7yyyQDuzBXyJu13%2BfJkOA3vgadmObcwfCgEh2Xkz6imgcY1ScBAYh%2Frblj6D1gF0dzKcA6afQss1IiNhjEAhm0jsaeEbxh7x7WmvaG6JhPDNY3l8WKuMz1sAldBeuFXz0WDndAnCiNeu2uAavwf)

**Reply by colevandersWands** [reply]
![shipit](https://qph.fs.quoracdn.net/main-qimg-c8781a4bb1f17e330b50cb35f851da05-c)
